packages/office-ui-fabric-react/src/components/Panel/examples/Panel.Large.Example.tsx

```diff
@@ -1,37 +1,33 @@
-import * as React from 'react';
 import { DefaultButton } from 'office-ui-fabric-react/lib/Button';
 import { Panel, PanelType } from 'office-ui-fabric-react/lib/Panel';
+import * as React from 'react';
 
-export class PanelLargeExample extends React.Component<
-  {},
-  {
-    showPanel: boolean;
-  }
-> {
-  constructor(props: {}) {
-    super(props);
-    this.state = { showPanel: false };
-  }
+interface IState {
+  showPanel: boolean;
+}
+
+export class PanelLargeExample extends React.Component<{}, IState> {
+  // tslint:disable-next-line:member-access
+  state = {
+    showPanel: false
+  };
 
-  public render(): JSX.Element {
+  public render() {
     return (
       <div>
-        <DefaultButton
-          secondaryText="Opens the Sample Panel"
-          // tslint:disable-next-line:jsx-no-lambda
-          onClick={() => this.setState({ showPanel: true })}
-          text="Open Panel"
-        />
-        <Panel
-          isOpen={this.state.showPanel}
-          // tslint:disable-next-line:jsx-no-lambda
-          onDismiss={() => this.setState({ showPanel: false })}
-          type={PanelType.large}
-          headerText="Large Panel"
-        >
+        <DefaultButton secondaryText="Opens the Sample Panel" onClick={this._onShowPanel} text="Open Panel" />
+        <Panel isOpen={this.state.showPanel} onDismiss={this._onDismissPanel} type={PanelType.large} headerText="Large Panel">
```

Comment:
Refactoring out the arrow functions, React will likely give a warning in the output for performance here if using arrow functions.